iceoryx_examples/icedelivery_on_c/README.md

```diff
@@ -0,0 +1,155 @@
+# [NOT YET IMPLEMENTED] icedelivery in C - Transfer data between POSIX applications
+
+**Hint** This example shows a work in progress. The API for the `publisher` and 
+`subscriber` is finished but we still have to integrate the new building blocks
+into RouDi. Till then whis C API will not work.
+
+You can find a more detailled description of the C API in the [iceoryx_binding_c README.md](../../iceoryx_binding_c)
+
+## The icedelivery example
+
+The behavior and structure is identical to the [icedelivery C++ example](../icedelivery/) 
+so that we explain here only the C API differences and not the 
+underlying mechanisms.
+
+### Subscriber
+
+Like in the 
+[icedelivery C++ example](../icedelivery/)
+we again follow the steps like:
+ 
+ 1. Create runtime instance.
+ 2. Create subscriber port.
+ 3. Subscribe to the offered service 
+ 4. Receive data 
+ 5. Unsubscribe.
+ 6. **C API: Additionally, we have to remove the Subscriber port!**
```

Comment:
I don't get this point. Can you elaborate?

Reply:
I guess this is because we do not have RAII in C